pkg/controller/cassandradatacenter/controller/jvm.go

```diff
@@ -0,0 +1,86 @@
+package controller
```

Comment:
Any reason why this is explicitly split into its own file? The ConfigMap contains everything related to Cassandras config, including JVM settings.